### PR TITLE
Transfer Owner Validations + Bug Fixes

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13952

*Description of changes:*
* Ties in HomeOwner Validations to Transfer
* Minor Bug fixes
* Ux Updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -366,6 +374,7 @@ export default defineComponent({
 
     const scrollToFirstError = async () => {
       setTimeout(() => {
+        document.getElementsByClassName('border-error-left').length > 0 &&
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue:377]: Safety check, was throwing errors once `document.getElementsByClassName('border-error-left')` was empty/errors cleared.

File: ppr-ui/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "lint": "vue-cli-service lint",
     "lint:no-fix": "vue-cli-service lint --no-fix",
     "clear_jest": "jest --clearCache",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
+    "test:unit": "vue-cli-service test:unit --testPathPattern",
```
Comment (dimak1) [inline ppr-ui/package.json]: No longer needed?

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:377]: Yea, I was hoping a timeout would prevent it, but this is another useful check.

**Review by owillborn — APPROVED** [review]
Looks solid

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue [hunk 3]
```diff
@@ -269,7 +285,18 @@ export default defineComponent({
       reviewed: false,
       isEditingMode: computed((): boolean => localState.currentlyEditingHomeOwnerId >= 0),
       isAddingMode: computed((): boolean => props.isAdding),
-      showTableError: computed((): boolean => showGroups.value && (hasMinimumGroups() || hasEmptyGroup.value)),
+      showTableError: computed((): boolean => {
+        return showGroups.value &&
+          (
+            props.validateTransfer &&
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Would still work for Registrations as expected?

**Review by dimak1 — APPROVED** [review]
Nice, looks good.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue (hunk 3 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: ```props.validateTransfer``` being false in a registration might give an incorrect error...

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Good catch guys, updated*!  🙏

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Works as expected (Now 😁)

**Pull request merged**